CODE_GENERATION.md

```diff
@@ -51,7 +51,12 @@ deleted. So you may want to delete everything in `apis/<serviceid>/v1alpha1` exc
 `generator-config.yaml` and then re-run the command.
 
 > If `apis/<serviceid>` is created from scratch, please add the new service name
```

Comment:
```suggestion
If `apis/<serviceid>` is created from scratch, please add the new service name
```

The block-quote is actually hiding, and not helping the visibility of this instruction. If we are attempting to hi-light this, perhaps *bold* would be more helpful?

Reply:
removed it and added headline

Reply:
@haarchri apologies for the heavy nitpicks here, but the header makes it a bit confusing because the next paragraph references the operation before the `## Makefile header`. Mind just stripping the header out?